Session garbage collection configuration (#8053)
magento/devdocs: src/_includes/config/locate-session.md

### `php.ini` example {#session-where-phpini}

As a user with `root` privileges, open your `php.ini` file and search for the value of `session.save_path`. This identifies where sessions are stored.

## Garbage collection configuration {#session-gc}

> To clean up expired sessions Magento application calls `gc` (_garbage collection_) handler randomly according to probability which is calculated by `gc_probability / gc_divisor`. For example, if we set these directives to `1/100` respectively, it means a probability of `1%` (_probability of one call of garbage collection per 100 requests_).

Reviewer: ```suggestion
To clean up expired sessions, Magento calls the `gc` (_garbage collection_) handler randomly according to a probability that is calculated by the `gc_probability / gc_divisor` directive. For example, if you set these directives to `1/100` respectively, it means a probability of `1%` (_probability of one call of garbage collection per 100 requests_).
```

> The garbage collection handler uses the `gc_maxlifetime` directive - the number of seconds after which the sessions will be seen as _garbage_ and potentially cleaned up.

Reviewer: ```suggestion
The garbage collection handler uses the `gc_maxlifetime` directive—the number of seconds after which the sessions will be seen as _garbage_ and potentially cleaned up.
```

> On some operating systems (Debian/Ubuntu) `session.gc_probability` set to `0` by default, and the garbage collection handler will never be executed.

Reviewer: ```suggestion
On some operating systems (Debian/Ubuntu), the default `session.gc_probability` directive is `0`, which prevents the garbage collection handler from running.
```

> `session.gc_` directives from the `php.ini` file can be overwritten in `<magento_root>/app/etc/env.php`:

Reviewer: ```suggestion
You can overwrite the `session.gc_` directives from the `php.ini` file in the `<magento_root>/app/etc/env.php` file:
```

```php
 'session' =>
    array (
      'save' => 'db',
      'gc_probability' => 1,
      'gc_divisor' => 1000,
      'gc_maxlifetime' => 1440
 ),
```

> The configuration depends on the traffic and specific needs of a certain Magento application.

Reviewer: ```suggestion

The configuration depends on the traffic and specific needs of a certain Magento application.
```
